src/main/java/net/floodlightcontroller/statistics/StatisticsCollector.java

```diff
@@ -244,7 +354,7 @@ public void init(FloodlightModuleContext context)
 
 		if (config.containsKey(INTERVAL_PORT_STATS_STR)) {
 			try {
-				portStatsInterval = Integer.parseInt(config.get(INTERVAL_PORT_STATS_STR).trim());
+				//portStatsInterval = Integer.parseInt(config.get(INTERVAL_PORT_STATS_STR).trim());
```

Comment:
Why'd you comment this out?

Reply:
It kept defaulting to the value set by the rest api